docs/resource-no-translation.md

```diff
@@ -0,0 +1,35 @@
+# resource-no-translation
+
+This rule ensures that each resource in a resource file has a translation. This rule
+looks to see if the source and target strings in a resource are different from
+each other. There are some exceptions to this where it does not report a problem if
+the two are the same:
+
+1. If the source locale and the target locale share a common language and script,
+then the translation can be the same as the source, as it is very common for
+dialects of a language to use the same words.
+1. If the source has the "do not translate" flag turned on.
+1. If the source string is a single word. To reduce the false positives, this rule
+only checks resources where there are multiple words in the source string.
```

Comment:
Add test that shows this scenario specifically (test name should be explicitly related)

Reply:
updated